.travis.yml

```diff
@@ -1,52 +0,0 @@
-language: python
```

Comment:
We should keep the Travis config file until all CI is migrated to Github actions (including Windows tests, readme link and removing Travis checks).

Reply:
Makes sense